modules/cluster-entitlements.adoc

```diff
@@ -117,6 +118,12 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-gcp"]
 :restricted:
 endif::[]
+ifeval::["{context}" == "installing-restricted-networks-alibaba-installer-provisioned"]
```

Comment:
We do not support installing an Alibaba cluster in a restricted environment. Did someone tell you to add this `ifeval`?

Reply:
@mjpytlak This edit was not my edit. Actually looks intentional, doesn't it?